ui/src/papers/index.tsx

```diff
@@ -69,6 +73,12 @@ interface CachedPaperWithMeta {
   paper: PaperWithMeta;
 }
 
+function cacheKey(paperId: string) {
+    // We append a version prefix so that we can bust the cache in clients when we release
+    // updates.
+    return `v0/${paperId}`; 
```

Comment:
Just to check my understanding, does this mean when we release updates, we would bump the version number for the full API (say to `v0.1`) in order to bust the cache?

Reply:
Yup, exactly. It's only necessary where there's a backwards incompatible change -- say, if the format of the data changes (like it did in this PR).